docs/src/content/docs/resources/community-content.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Community Content
+description: Discover community-produced guides, articles and videos to help you learn and build with Starlight!
+---
+
+:::tip[Add your own!]
+Have you produced content about Starlight?
+Open a PR adding a link to this page!
+:::
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+## Articles and Reviews
+
+Here is a collection of posts and articles to learn more about Starlight and other people's experiences:
+
+<CardGrid>
+	<LinkCard
+		href="https://devm.io/open-source/starlight-astro"
+		title="Static Site Generation with Starlight: “No idea is too big or too small when designing components”"
+		description="Interview with Chris Swithinbank, Starlight Lead"
+	/>
+	<LinkCard
+		href="https://frontendatscale.com/blog/hybrid-frontend-architecture/"
+		title="Hybrid Frontend Architecture with Astro and Starlight"
+		description="Discussion with Ben Holmes about application design and architecture and implementing a Starlight prototype"
```

Comment:
```suggestion
		description="Maxi Ferreira and Ben Holmes build and discuss a fully-featured docs site with Astro, Starlight, TinaCMS, and a personalized interactive API Playground with authentication."
```

Reply:
Nice changes/catches both of you, but in fact there should be no hyphen at all for fully featured!  :smile: 

It takes a village....